### ISSUE TITLE
Cross line tie never terminated with two voices.

### ISSUE THREAD

**Issue by johnkolen** [paulrosen/abcjs#697]
The tie at the end of measure 4 the melody voice should match with the first note of measure 5, but it never terminates (tie arc continuing down the page). It works as expected with only one voice, however.

```
X:1
T:Lawless Greenness
M:2/4
Q:1/4=105
V:melody clef=treble name=melody
V:harmony clef=bass name=harmony
K:GLyd
[V:melody]
d4-|d4|B4|  DD B,B,-|
[V:harmony]
 B,D Gz|^C^F Az|D^F Bz|B,E Gz|
[V:melody]
B,DE2|z2B2|G,2B,2|G,2^CA|
[V:harmony]
B,D Gz|A,D ^Fz|[G,B,D]4|  B,,E, G,z|
```
![Screen Shot 2021-06-12 at 5 50 42 PM](https://user-images.githubusercontent.com/653313/121792229-2d01c600-cba7-11eb-9f10-bbaed30a88c8.png)

![Screen Shot 2021-06-12 at 5 55 12 PM](https://user-images.githubusercontent.com/653313/121792249-62a6af00-cba7-11eb-91b7-66f81e0a22f0.png)

**Comment by paulrosen** [comment]
Thanks for the report - I fixed a few similar issues and I'll look at this.

**Comment by majcn** [comment]
Just wanted to post the same issue and found that it's already there.

is there any way to workaround it?

**Comment by paulrosen** [comment]
Thanks for the bump on this - it fell off my radar. A workaround is to separate the voices. For instance, with the code above, rearranging it to the following works.
```
X:1
T:Lawless Greenness
M:2/4
Q:1/4=105
V:melody clef=treble name=melody
V:harmony clef=bass name=harmony
K:GLyd
[V:melody]
d4-|d4|B4|  DD B,B,-|
B,DE2|z2B2|G,2B,2|G,2^CA|
[V:harmony]
 B,D Gz|^C^F Az|D^F Bz|B,E Gz|
B,D Gz|A,D ^Fz|[G,B,D]4|  B,,E, G,z|
```

**Comment by paulrosen** [comment]
Fixed in version 6.2.0
